samples/browser/src/main/java/org/mozilla/samples/browser/BaseBrowserFragment.kt

```diff
@@ -212,6 +235,7 @@ abstract class BaseBrowserFragment : Fragment(), UserInteractionHandler {
             REQUEST_CODE_DOWNLOAD_PERMISSIONS -> downloadsFeature.get()
             REQUEST_CODE_PROMPT_PERMISSIONS -> promptFeature.get()
             REQUEST_CODE_APP_PERMISSIONS -> sitePermissionsFeature.get()
+            REQUEST_CODE_P2P_PERMISSIONS -> p2pFeature
```

Comment:
```suggestion
            REQUEST_CODE_P2P_PERMISSIONS -> p2pIntegration.get()?.feature
```

It's not great, but you could reference the inner feature directly from the integration in order to remove this second reference to the feature.

```
p2pFeature = p2pIntegration.get()?.feature
```